bin/install.py

```diff
@@ -95,6 +95,7 @@ def coursier_bootstrap(target, main):
             "java", "-jar", BLOOP_COURSIER_TARGET, "bootstrap", BLOOP_ARTIFACT,
             "-r", "bintray:scalameta/maven",
             "-r", "bintray:scalacenter/releases",
+            "-r", "https://oss.sonatype.org/content/repositories/snapshots",
```

Comment:
`"-r", "sonatype:snapshots",`

Reply:
Will this work? I copy pasted this line from the coursier README.